qt/scientific_interfaces/Indirect/IndirectFitDataModel.h

```diff
@@ -88,6 +88,7 @@ class MANTIDQT_INDIRECT_DLL IndirectFitDataModel
   std::vector<double>
   getExcludeRegionVector(FitDomainIndex index) const override;
   std::string getExcludeRegion(FitDomainIndex index) const override;
+  void setExcludeRegion(const std::string &exclude, FitDomainIndex index);
```

Comment:
It might be worth adding a pure virtual method to the interface class as is done for `getExcludeRegion`